internal/testutils/dummy_identity.go

```diff
@@ -0,0 +1,45 @@
+package testutils
```

Comment:
I am trying to hate all utils/tools packages, can we try to figure out something better? I think that this could actually go into the ctxval package where I ask you to put the Get func..

Reply:
I do not like mixing production and testing helper code, but if we make it compile only with test flag and make sure it's clear from the name, I'm happy to do so, WDYT?